plugins/slack_webhooks/test/decorators/stage_decorator_test.rb

```diff
@@ -27,4 +27,32 @@
       stage.slack_webhooks.size.must_equal 0
     end
   end
+
+  describe "#slack_buddy_channels" do
+    it "returns channels when for_buddy is true" do
+      stage.slack_webhooks = [SlackWebhook.new(channel: "test", webhook_url: "http://slack.com/abcdef",
```

Comment:
yes

On Thu, Jun 23, 2016 at 10:26 AM, Zehua Liu <notifications@github.com>
wrote:

> In plugins/slack_webhooks/test/decorators/stage_decorator_test.rb
> <https://github.com/zendesk/samson/pull/1058#discussion_r68277564>:
>
> > @@ -27,4 +27,32 @@
> >        stage.slack_webhooks.size.must_equal 0
> >      end
> >    end
> > +
> > +  describe "#slack_buddy_channels" do
> > +    it "returns channels when for_buddy is true" do
> > +      stage.slack_webhooks = [SlackWebhook.new(channel: "test", webhook_url: "http://slack.com/abcdef",
>
> do you mean 2 space indent in the next line?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1058/files/1c0e3b4444e4a02023ed261f0c6443f9203994e5#r68277564>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ-OapmLOKaj12hNZs05UNR_Ss7Ppks5qOsHEgaJpZM4Iyxjs>
> .
>